### ISSUE TITLE
Background showing on bottom of media screen

### ISSUE THREAD

**Issue by DSHoyle** [AntonyCorbett/JwlMediaWin#10]
After installing Windows 10 update 1089 (October 2018 update) our media display is showing the pc background along the bottom of the screen, about the height of a taskbar. When JWL opens the display is ‘normal’ for about 5 to 10 seconds, then the line appears, this happens if you close then open JWL again, there’s always a few seconds of normal display then the background line appears.

We were having a problem with the year text vanishing after displaying an image with OnlyM and used the JWL media fix service to sort it out and it worked faultlessly.

OnlyM is running with JWL compatibility off and I’ve made sure that the extended desktop doesn’t display the taskbar.

If I disable the media fix and then close JWL and reopen it, the line goes away. We can use OnlyM in compatibility mode with the fix off and the year text is not vanishing (so far) but the transition is very messy again.  

Any idea why an update has caused this behaviour?

I’m away until the weekend, so will not be able to access the logs or add an image but will as soon as possible.

**Comment by Blann** [comment]
Sounds like the JWL media window is being resized incorrectly. Not sure how the resolution of the media display is evaluated – may be a bug. 

Windows updates often interfere with graphics drivers – have you tried rolling back the driver to what you had before and seeing if that has any effect?

**Comment by DSHoyle** [comment]
The driver was fine, but it seems the update did an annoying random thing to the display settings. In display properties there is an option to change the size of text apps and other items (Scale and Layout) like this 
https://goo.gl/images/nv48hh

The recommended setting was 150 percent, when I set it back to 100 percent the blue line was gone. The update must have changed it for all users as I had to change it for the 4 user accounts individually.

Glad we are back to the smooth transition between OnlyM and JWL!
